onmt/modules/__init__.py

```diff
@@ -1,16 +1,20 @@
 from onmt.modules.Util import LayerNorm, Bottle, BottleLinear, \
     BottleLayerNorm, BottleSoftmax, aeq
 from onmt.modules.GlobalAttention import GlobalAttention
+from onmt.modules.ConvAttention import ConvAttention
 from onmt.modules.ImageEncoder import ImageEncoder
 from onmt.modules.CopyGenerator import CopyGenerator, CopyCriterion
 from onmt.modules.StructuredAttention import MatrixTree
 from onmt.modules.Transformer import TransformerEncoder, TransformerDecoder
+from onmt.modules.Conv2Conv import ConvEncoder, ConvDecoder
 from onmt.modules.MultiHeadedAttn import MultiHeadedAttention
 from onmt.modules.StackedRNN import StackedLSTM, StackedGRU
+from onmt.modules.WeightNorm import WN_Conv2d
```

Comment:
Can you give this a slightly better name. Maybe spell out WeightNormConv2D

Reply:
done